models/User.js

```diff
@@ -85,8 +85,12 @@ class User {
     return db.query(sql, [id], instanceFromRowUser).then(([user]) => user);
   }
 
-  static findAll() {
-    const sql = 'SELECT * FROM users';
+  static indexMembers() {
+    const sql = `
+      SELECT * FROM users
+      WHERE NOT spotId IS NULL
```

Comment:
Should be checking on role. If a user doesn't have a spot, then they won't show up